admin-ui/plugins/auth-server/components/AuthN/LdapForm.js

```diff
@@ -0,0 +1,209 @@
+          name="defaultAuthnMethod"
+          formik={formik}
+          value={formik.values.defaultAuthnMethod}
+          handler={() =>
+            formik.setFieldValue('defaultAuthnMethod', !formik.values.defaultAuthnMethod)
+          }
+        />
+        <GluuInputRow
+          label={STRINGS.authn.ldap.fields.bind_dn}
+          name="bindDN"
+          value={formik.values.bindDN}
+          formik={formik}
+          onChange={formik.handleChange}
+          onBlur={formik.handleBlur}
+          required
+          placeholder={STRINGS.authn.ldap.placeholders.bind_dn}
+          error={formik.touched.bindDN && formik.errors.bindDN}
+        />
+        <GluuInputRow
+          label={STRINGS.authn.ldap.fields.max_connections}
+          name="maxConnections"
+          value={formik.values.maxConnections}
+          formik={formik}
+          onChange={formik.handleChange}
+          onBlur={formik.handleBlur}
+          required
+          placeholder={STRINGS.authn.ldap.placeholders.max_connections}
+          error={formik.touched.maxConnections && formik.errors.maxConnections}
+        />
+        <GluuInputRow
+          label={STRINGS.authn.ldap.fields.local_primary_key}
+          name="localPrimaryKey"
+          value={formik.values.localPrimaryKey}
+          formik={formik}
+          onChange={formik.handleChange}
+          onBlur={formik.handleBlur}
+          required
+          placeholder={STRINGS.authn.ldap.placeholders.local_primary_key}
+          error={formik.touched.localPrimaryKey && formik.errors.localPrimaryKey}
+        />
+        <GluuInputRow
+          label={STRINGS.authn.ldap.fields.remote_primary_key}
+          name="primaryKey"
+          value={formik.values.primaryKey}
+          formik={formik}
+          onChange={formik.handleChange}
+          onBlur={formik.handleBlur}
+          required
+          placeholder={STRINGS.authn.ldap.placeholders.remote_primary_key}
+          error={formik.touched.primaryKey && formik.errors.primaryKey}
+        />
+        <GluuInputRow
+          label={STRINGS.authn.ldap.fields.remote_ldap_server}
+          name="servers"
+          value={formik.values?.servers}
+          formik={formik}
+          onChange={(e) => {
+            const arr = e.target.value
+              .split(',')
+              .map((s) => s.trim())
+              .filter(Boolean)
+            formik.setFieldValue('servers', arr.length ? arr : [''])
+          }}
+          onBlur={formik.handleBlur}
+          required
+          placeholder={STRINGS.authn.ldap.placeholders.remote_ldap_server}
+          error={
+            formik.touched.servers &&
+            (Array.isArray(formik.errors.servers)
+              ? formik.errors.servers.join(', ')
+              : formik.errors.servers)
+          }
+        />
+        <GluuInputRow
+          label={STRINGS.authn.ldap.fields.base_dns}
+          name="baseDNs"
+          value={formik.values.baseDNs}
```

Comment:
Ensure consistent handling of baseDNs field

Similar to the servers field, baseDNs should be consistently handled as an array.



```diff
-          value={formik.values.baseDNs}
+          value={Array.isArray(formik.values.baseDNs) ? formik.values.baseDNs.join(', ') : ''}
```